snp_motion_planning/src/planning_server.cpp

```diff
@@ -114,36 +123,36 @@ class PlanningServer
     plotter_ = std::make_shared<tesseract_rosutils::ROSPlotting>(env_->getRootLinkName());
 
     // Create the environment monitor
-    tesseract_monitor_ =
-        std::make_shared<tesseract_monitoring::EnvironmentMonitor>(node_, env_, TESSERACT_MONITOR_NAMESPACE);
-    tesseract_monitor_->startStateMonitor(tesseract_monitoring::DEFAULT_JOINT_STATES_TOPIC, false);
-
-    // Register custom process planners
-    planning_server_->registerProcessPlanner(TRANSITION_PLANNER, createTransitionTaskflow());
-    planning_server_->registerProcessPlanner(FREESPACE_PLANNER, createFreespaceTaskflow());
-    planning_server_->registerProcessPlanner(RASTER_PLANNER, createRasterTaskflow());
-
+    tesseract_monitor_ = std::make_shared<tesseract_monitoring::ROSEnvironmentMonitor>(internal_node_, env_,
+                                                                                       TESSERACT_MONITOR_NAMESPACE);
+    tesseract_monitor_->setEnvironmentPublishingFrequency(30.0);
+    tesseract_monitor_->startPublishingEnvironment();
+    tesseract_monitor_->startStateMonitor("/robot_joint_states", true);
+
+    //    // Register custom process planners
+    //    planning_server_->registerProcessPlanner(TRANSITION_PLANNER, createTransitionTaskflow());
+    //    planning_server_->registerProcessPlanner(FREESPACE_PLANNER, createFreespaceTaskflow());
+    //    planning_server_->registerProcessPlanner(RASTER_PLANNER, createRasterTaskflow());
+
+    // TODO: PLANNING SEEMS TO FAIL
```

Comment:
Is this still true? If so it would be preferable to get successful planning before merging this PR

Reply:
Oh oops, that was from me working the other day, it works now.